### DISCUSSION TITLE
/nsm/pcap fills up quickly

### DISCUSSION THREAD

**Discussion by ngocnn97** [Security-Onion-Solutions/securityonion#8524] [Unsupported Versions] [answered]
Hi

I installed SO 2.3.130 standalone. I am having a problem that /nsm/pcap fills up very fast with 6Mb/minute.
Can someone help me explain what the files in /nsm/pcap are and what they are used for?

Output of ```sudo du --max-depth=1 -h /nsm``` :

4.9G    /nsm/docker-registry
1.3G    /nsm/repo
615M    /nsm/backup
11G     /nsm/elasticsearch
79G     /nsm/pcap
0       /nsm/pcaptmp
0       /nsm/pcapout
627M    /nsm/pcapindex
46M     /nsm/suricata
0       /nsm/soc
249M    /nsm/mysql
383M    /nsm/influxdb
1.7M    /nsm/grafana
24K     /nsm/osquery
2.0G    /nsm/wazuh
276K    /nsm/strelka
0       /nsm/import
4.0K    /nsm/logstash
99G     /nsm

**Comment by isaacgolding** [comment, marked as answer]
In SO your sensor nic captures all of the network traffic it sees and this is turned into what are called PCAP files.   These pcap files (and logfiles) will fill up the NSM by design.   At around 80-90% the system will start to automatically purge older PCAPS and other files from the NSM to make room for newer ones.    This is why the hardware requirements for drive space are so high.    My install is on a 2 TB drive and the NSM fills up in around 3 weeks from a fresh install.      If the NSM goes to 100% something is wrong but 80-90% is ok.

**Reply by ngocnn97** [reply]
Thanks a lot. I scaled down /nsm and watched, just as you described, /nsm just stopped at 90% and rotated after that
